controllers/microservices_controller.go

```diff
@@ -100,6 +108,28 @@ func (r *MicroservicesReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	return ctrl.Result{}, nil
 }
 
+func (r *MicroservicesReconciler) getStorageConfig(ctx context.Context, tempo v1alpha1.Microservices) (*manifests.StorageParams, error) {
+	storageSecret := &corev1.Secret{}
+	err := r.Get(ctx, types.NamespacedName{Namespace: tempo.Namespace, Name: tempo.Spec.Storage.Secret}, storageSecret)
+	// TODO check if secret exists in the validating webhook
```

Comment:
I assume at this point in our dev cycle TODOs are still ok?

Reply:
yea, I will book a ticket for the webhook

Reply:
https://github.com/os-observability/tempo-operator/issues/58